Update C++ rules (#3203)
SonarSource/rspec: rules/S6197/cfamily/rule.adoc

== Why is this an issue?

> {cpp}20 introduces the ranges library. A range is a group of items that can be iterated over. It should provide a ``++begin++`` iterator and an ``++end++`` sentinel. As you can guess, all the STL containers are ranges.

Reviewer: ```suggestion
{cpp}20 introduces the ranges library. A range is a group of items that can be iterated over. It should provide a ``++begin++`` iterator and an ``++end++`` sentinel. All the existing STL containers are ranges.
```
